### DISCUSSION TITLE
Difference between two readings

### DISCUSSION THREAD

**Discussion by bentcoder** [arl/statsviz#116] [Q&A]
Hi,

I've run two tests and the outcome is mostly same apart from three graphs (marked within green boxes). Based on what I understand from the descriptions for the graphs, the image on top (marked with 1) should be considered better. Am I reading it right?

Thanks.

![db-pointer](https://github.com/arl/statsviz/assets/8837278/46c8fdd0-902c-4160-92a1-52e34ea7e263)
---
![db-copy](https://github.com/arl/statsviz/assets/8837278/4668c34e-c637-40ca-ae82-7a564b72b025)

**Comment by arl** [comment]
Hi @bentcoder 

Something doesn't look right with the first screenshot. It seems that there is no data. 
For example, the 'completed GC cycles' plot should show vertical bars when GC happens, and we can clearly see that garbage collections did happen (from the dotted vertical lines appearing on many other plots).

Depending on the Go version you're using, some metrics are absent because not exported from `runtime/metrics`.

Were you using different go versions for the tests? 
Can you reproduce this?

Thank you
